client-ui/src/app/model/lineage-graph/leneage-graph-legend.models.ts

```diff
@@ -15,26 +15,31 @@
  */
 
 
-import { LINE_COLOR_HLT_IMPACT, LINE_COLOR_HLT_LINEAGE, LINE_COLOR_HLT_PRIMARY } from './leneage-graph.models';
+import {LINE_COLOR_HLT_IMPACT, LINE_COLOR_HLT_LINEAGE, LINE_COLOR_HLT_PRIMARY} from './leneage-graph.models';
 
 
-export interface LineageGraphLegend {
+export type LineageGraphLegend = {
   title: string;
   color: string;
   description: string;
 }
 
-export function getLineageGraphLegend(): LineageGraphLegend[] {
-  return [
-    {
+export type LineageGraphLegendKeys =
+  "Usage" |
+  "Lineage" |
+  "Impact"
+
+export const LineageGraphLegends: { [key in LineageGraphLegendKeys]: LineageGraphLegend } =
```

Comment:
We did not disscuss nameing convention yet, but still the current name format of the const means for me that this is a class or interface. Global const definition is UPPER_SNACK_CASE.

Reply:
Yeah, this is a CQC topic. Let's leave it for now until we make a statement.